statusboard/src/components/Header/Header.js

```diff
@@ -9,15 +9,15 @@ function Header() {
   // https://dequeuniversity.com/assets/html/jquery-summit/html5/slides/headings.html
   return (
     <header className="App-header">
-      <NavLink to="/" className="home-link">
+      <NavLink to="https://brigade.codeforamerica.org/" className="home-link">
```

Comment:
I see your point. That Brigade Network Home Page has a link to us, so I thought in one way we are making even more "official" that this is a CfA tool. It could be that even CfA leadership don't like this linkage...
But has it is now it is useless, so I'm thinking to perhaps remove any link from the logo.